src/tests/ftest/pool/verify_space.py

```diff
@@ -0,0 +1,347 @@
+        pool_size (list): list of pool_size dictionaries
+
+    Returns:
+        bool: is all of the pool size is greater than or equal to the available
+    """
+    current_size = human_to_bytes(pool_size[-1]['data'][rank]['size'])
+    current_avail = human_to_bytes(pool_size[-1]['data'][rank]['avail'])
+    return current_size >= current_avail
+
+
+def compare_equal(rank, pool_size):
+    """Determine if the previous value equal the current value.
+
+    Args:
+        rank (int): server rank
+        pool_size (list): list of pool_size dictionaries
+
+    Returns:
+        bool: does the previous value equal the current value
+    """
+    previous_avail = human_to_bytes(pool_size[-2]['data'][rank]['avail'])
+    current_avail = human_to_bytes(pool_size[-1]['data'][rank]['avail'])
+    return previous_avail == current_avail
+
+
+def compare_reduced(rank, pool_size):
+    """Determine if the previous value is greater than the current value.
+
+    Args:
+        rank (int): server rank
+        pool_size (list): list of pool_size dictionaries
+
+    Returns:
+        bool: does the previous value equal the current value
+    """
+    previous_avail = human_to_bytes(pool_size[-2]['data'][rank]['avail'])
+    current_avail = human_to_bytes(pool_size[-1]['data'][rank]['avail'])
+    return previous_avail > current_avail
+
+
+class VerifyPoolSpace(TestWithServers):
+    """Verify pool space with system commands.
+
+    :avocado: recursive
+    """
+
+    def _query_pool_size(self, description, pools):
+        """Query the pool size for the specified pools.
+
+        Args:
+            description (str): pool description
+            pools (list): list of pools to query
+        """
+        self.log_step(f'Query pool information for {description}')
+        for pool in pools:
+            pool.query()
+
+    def _create_pools(self, description, namespaces):
+        """Create the specified number of pools.
+
+        Args:
+            description (str): pool description
+            namespaces (list): pool namespaces
+
+        Returns:
+            list: a list of created pools
+        """
+        pools = []
+        self.log_step(' '.join(['Create', description]), True)
+        for item in namespaces:
+            namespace = os.path.join(os.sep, 'run', '_'.join(['pool', 'rank', str(item)]), '*')
+            pools.append(self.get_pool(namespace=namespace))
+        self._query_pool_size(description, pools)
+        return pools
+
+    def _write_data(self, description, ior_kwargs, container, block_size):
+        """Write data using ior to the specified pool and container.
+
+        Args:
+            description (str): pool description
+            ior_kwargs (dict): arguments to use to run ior
+            container (TestContainer): the container in which to write data
+            block_size (str): block size to use with the ior
+        """
+        self.log_step(f'Writing data ({block_size} block size) one of {description}')
```

Comment:
Is there a typo in this message? I don't understand what it's trying to say at then end

Reply:
Fixed, thanks.